File: Dragon-WebApi/Models/CampaignMessage.cs
namespace Dragon_WebApi.Models
{
    public class CampaignMessage
    {
        public int Id { get; set; }
        public int CampaignId { get; set; }
        public int UserId { get; set; }
        public string UserName { get; set; }
        public string MessageText { get; set; }
        public bool IsPrivate { get; set; }
        public DateTime CreationDate { get; set; }

        public CampaignMessage()
        {
            Id = 0;
            CampaignId = 0;
            UserId = 0;
            UserName = "";
            MessageText = "";
            IsPrivate = false;
            CreationDate = new DateTime();
        }
    }
}

File: people-among-dragons/src/models/campaign-message.ts
export class CampaignMessage {
    _messageId: number;
    _campaignId: number;
    _userId: number;
    _userName: string;
    _messageText: string;
    _isPrivate: boolean;
    _creationDate: Date;

    constructor(messageId: number, campaignId: number, userId: number, userName: string, messageText: string, isPrivate: boolean,  creationDate: Date){
        this._messageId = messageId;
        this._campaignId = campaignId;
        this._userId = userId;
        this._userName = userName;
        this._messageText = messageText;
        this._isPrivate = isPrivate;
        this._creationDate = creationDate;
    }
}